### ISSUE TITLE
Support Adaptive Icons

### ISSUE THREAD

**Issue by MarkOSullivan94** [franzsilva/flutter_launcher_icons#23]
One of the suggestions I've seen on Reddit (thanks conman__1040!) is to implement adaptive icons for the Flutter Android app. 

Apparently the default icon is adaptive and so investigation will need to be done to see how it's done and how we can replicate the behavior for new icons.

**Comment by gedw99** [comment]
just for reference as this aspect was new to me:
https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive

**Comment by markmooibroek** [comment]
Think its pretty straightforward to add this as a functionality that supports a solid color as background with a png (with alpha) for the overlay. I use that approach in 95% of the apps i create for my customers: 

```xml
<?xml version="1.0" encoding="utf-8"?>
<adaptive-icon xmlns:android="http://schemas.android.com/tools">
    <background android:drawable="@color/ic_background"/>
    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
</adaptive-icon>
```

Maybe its an idea to add to 2 new params in the settings that define the fore and background. Then we can use the existing `image_path` to generate all pre API 26 fallback icons.

```yml
flutter_icons:
  image_path: "icon/icon.png" 
  android: true
  android_adaptive_background: #FF990099
  android_adaptive_foreground: "icon_foreground.png"
  ios: "Example-Icon"
```

Just my 2 cents, Keep up the great work 👍

**Comment by MarkOSullivan94** [comment]
Implemented by @markmooibroek in #28. Thanks @markmooibroek!
